routes/clucks.js

```diff
@@ -0,0 +1,72 @@
+const express = require('express');
+const knex = require('../db/client');
+const router = express.Router();
+
+//-------------- Index to List Clucks
+router.get('/',(req,res) =>{
+
+    const faker = require('faker');
+    const randomImage = faker.image.nature(50,50,true);
+
+    knex('cluckr')
+    .orderBy('created_at','desc')
+    .then(clucks =>{
+        res.render('clucks/index',{clucks:clucks,randomImage:randomImage})
+    })
+})
+
+//-------------- Create New Clucks
+router.get('/new',(req,res) =>{
+    res.render('clucks/new', {cluck:false})
+})
+
+
+router.post('/',(req,res) =>{
+    knex('cluckr')
+    .insert({
+        content: req.body.content,
+        image_url: req.body.image_url,
+        username: req.cookies.username
+    })
+    .returning('*')
+    .then(clucks =>{
+        const cluck = clucks[0];
+        res.redirect('/clucks')
+    })
+})
+
+
+friendlyTime = function (previous) {
```

Comment:
declare variable? It's really not suggested to leave your variable global like this, especially in a large enough codebase.